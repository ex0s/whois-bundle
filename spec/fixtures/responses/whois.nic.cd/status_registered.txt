Domain Name: google.cd
Domain ID: 5758-CD
Sponsoring Registrar: MARKMONITOR
Sponsoring Registrar IANA ID: 






DNSSEC: unsigned


TERMS OF USE: You are not authorized to access or query our WHOIS database through the use of electronic processes that are high-volume and automated. This WHOIS database is provided by CDnic as a service to the internet community on behalf of Congo, the Democratic Republic (La Republique Democratique du Congo ? RDC). 

The data is for information purposes only. CDnic does not guarantee its accuracy. By submitting a WHOIS query, you agree to abide by the following terms of use: You agree that you may use this Data only for lawful purposes and that under no circumstances will you use this Data to: (1) allow, enable, or otherwise support the transmission of mass unsolicited, commercial advertising or solicitations via e-mail, telephone, or facsimile; or (2) enable high volume, automated, electronic processes that apply to CDnic it's members (or CDnic or member computer systems). The compilation, repackaging, dissemination or other use of this Data is expressly prohibited.


>>> Last update of WHOIS database: 2014-02-18T11:46:50.985Z <<<
