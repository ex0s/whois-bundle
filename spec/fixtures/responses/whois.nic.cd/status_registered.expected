#domain
  %s == "google.cd"

#domain_id
  %s == "5758-CD"


#status
  %s == :registered

#available?
  %s == false

#registered?
  %s == true


#created_on
  %s == nil

#updated_on
  %s == nil

#expires_on
  %s == nil


#registrar
  %s %CLASS{registrar}
  %s.id           == nil
  %s.name         == "MARKMONITOR"
  %s.organization == nil
  %s.url          == nil


#nameservers
  %s %CLASS{array}
  %s %SIZE{0}
